query-request.hh

```diff
@@ -303,6 +303,8 @@ public:
     std::optional<query::max_result_size> max_result_size;
     uint32_t row_limit_high_bits;
     api::timestamp_type read_timestamp; // not serialized
+    sstring query_string;
```

Comment:
I don't think this is a good idea. A query string is not at hand for most queries, as they are prepared and those don't have the raw query string anymore. We also support more than a single protocol. @cmm is working on an alternative way to associate errors/warnings with a certain query.

Reply:
We also have a query_uuid across replicas. But it would be harder to find query text by uuid in coordinator logs. And we need to send back to coordinator flag to log particular query text with query_uuid.